src/command_modules/azure-cli-sf/setup.py

```diff
@@ -35,7 +35,7 @@
     'azure-servicefabric==5.6.130',
```

Comment:
We want to specifically pin this because we don't want to users to auto-upgrade at this time. I know we have pending work to move into our own CLI, so for the time being I'd just like to leave this as-is, since this version won't change until then anyways. Lets discuss over email what's the right thing to do with others.